app/controllers/Api.scala

```diff
@@ -169,6 +169,15 @@ trait Api extends Logging {
     singleItem(Prism.instanceAgent, arn)
   }
 
+  // for some bizarre compilation reason this can't be called 'lambdaList'
+  def functionList = Action.async { implicit request =>
+    itemList(Prism.lambdaAgent, "lambdas")
+  }
+  // for some bizarre compilation reason this can't be called 'lambda'
```

Comment:
If we think `function` isn't intention revealing enough, would `getSingleLambda` work?

Reply:
done https://github.com/guardian/prism/pull/71/commits/1de2c079bd88d8ef0134c6811e832f65d0313af3